index.html

```diff
@@ -7,31 +7,40 @@
     <meta name="description" content="Abdul Ali Noori's portfolio" />
     <link rel="stylesheet" href="styles.css">
     <link rel="stylesheet" href="desktop-styles.css">
+    
     <title>Portfolio</title>
 </head>
 <body>
     <div class="wrapper">
         <header>
             <nav class="navbar">
-
-                <a href="#headline" class="welcome-logo">Noori</a>
-
-                <ul class="menu">
-                    <li><a class="menuItem" href="#">Portfolio</a></li>
-                    <li><a class="menuItem" href="#">Abbout</a></li>
-                    <li><a class="menuItem" href="#">Contact</a></li>
+    
+                <a href="#" class="nav-branding">Noori</a>
+    
+                <ul class="nav-menu">
+    
+                    <li class="nav-item">
+                        <a class="nav-link" href="#recent-work">Portfolio</a>
+                    </li>
+    
+                    <li class="nav-item">
+                        <a class="nav-link" href="#about-me">About</a>
+                    </li>
+                    
+                    <li class="nav-item">
+                        <a class="nav-link" href="#from-section">Contact</a>
```

Comment:
- [ ] Good job 😃 at implementing the functionality of the mobile menu, but I see here the font size of the links of the mobile menu is not like the Figma Design it is less than the Figma, so please try to stick to the font at the Figma
- [ ] Als consider 😊adding the color to the link to be like the Figma Design as I see you didn't add color to the links, so kindly use the one in the Figma and with that, the Design will be great like the Figma💯

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/217322774-ce683f40-f45c-4341-92b8-9b450e15deb8.png)   |    ![image](https://user-images.githubusercontent.com/83360636/217322735-9524bcdc-0ee2-4ff4-8ff6-7408e80f3869.png) |